doc/nrf/ug_partition_manager.rst

```diff
@@ -279,3 +279,38 @@ CMake usage
 
       --slot-size $<TARGET_PROPERTY:partition_manager,MCUBOOT_SLOT_SIZE>
 
+
+Predefined configuration
+========================
+Predefined configuration is a feature intended for deployed products which consists of multiple images, but where
+only a subset of the included images will be upgrade-able through firmware update mechanisms.
+In this case, it is important that the images to be upgraded are linked to the same address as what is on the existing product.
```

Comment:
For clarity:
"In this case" -> "In this use case"

Reply:
Made it "In these use cases"